src/controller/net_controller.c

```diff
@@ -1156,11 +1135,14 @@ void menu_controller_hook(controller *ctrl, int action) {
         ENetPacket *packet;
         serial_create(&ser);
         serial_write_int8(&ser, EVENT_TYPE_ACTION);
+        // blank header
```

Comment:
Issue with this style is that we have no field names. What is this value normally ?

Reply:
look at the other places ACTION is encoded?

Reply:
Sure. But my main point is that it would be nice if we had some sort of natural documentation for these. I wonder if there could be some sort of 
```
send_action(action_dto dto) {
    serial ser;
    serial_write_xxx(ser, dto->field1);
    ....
    send(ser);
}
```

... or something.

Reply:
I should really just have 2 kinds of event messages.